config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -117,7 +117,7 @@ periodics:
       - --test-cmd-args=--testoverrides=./testing/overrides/5000_nodes.yaml
       - --test-cmd-args=--testoverrides=./testing/load/golang/custom_api_call_thresholds.yaml
       - --test-cmd-name=ClusterLoaderV2
-      - --timeout=180m
+      - --timeout=210m
```

Comment:
Please also bump the `.decoration_config.timeout` field value by 30 mins.

Reply:
Right, we need more time in prow for TearDown cleanup. Done, increased.